lib/datadog/tracing/contrib/sinatra/tracer_middleware.rb

```diff
@@ -111,10 +120,6 @@ def analytics_sample_rate
           def configuration
             Datadog.configuration.tracing[:sinatra]
           end
-
-          def header_to_rack_header(name)
```

Comment:
Not used.

Reply:
Just double checking here...

https://github.com/DataDog/dd-trace-rb/pull/2217/files#diff-eb65416700be66ce2d6531a89dd8510475f7bdfe0bb1037e639a11e9a632ef50R48 calls `request_header_tags` which calls `header_to_rack_header(header)`.

This method was duplicated, and this duplication is not in use. Is that correct?